src/bgnbd_expectation.cpp

```diff
@@ -0,0 +1,82 @@
+#include <RcppArmadillo.h>
+#include <math.h>
+#include "bgnbd_expectation.h"
+#include "clv_vectorized.h"
+
+//' @name bgnbd_expectation
+//' @title BG/NBD: Unconditional Expectation
+//'
+//' @description Computes the expected number of repeat transactions in the interval (0, vT_i]
+//' for a randomly selected customer, where 0 is defined as the point when the customer came alive.
+//'
+//' @template template_params_bgnbd
+//' @param vT_i Number of periods since the customer came alive
```

Comment:
move @param vT_i docu to template, re-use in all expectation functions, incl ggomnbd